$OpenBSD: patch-deps_ntlmclient_crypt_openssl_c,v 1.1 2021/11/19 08:01:06 ajacoutot Exp $

deps/ntlmclient/crypt_openssl.c:49:20:
error: static declaration of 'HMAC_CTX_cleanup' follows non-static declaration

Index: deps/ntlmclient/crypt_openssl.c
--- deps/ntlmclient/crypt_openssl.c.orig
+++ deps/ntlmclient/crypt_openssl.c
@@ -44,7 +44,8 @@ static inline void HMAC_CTX_free(HMAC_CTX *ctx)
 
 #endif
 
-#if OPENSSL_VERSION_NUMBER >= 0x10100000L || defined(CRYPT_OPENSSL_DYNAMIC)
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L || defined(CRYPT_OPENSSL_DYNAMIC)) \
+	&& !defined(LIBRESSL_VERSION_NUMBER)
 
 static inline void HMAC_CTX_cleanup(HMAC_CTX *ctx)
 {
